KarhooUISDK/Classes/Payment/AddCard/PaymentFactory.swift

```diff
@@ -32,4 +33,8 @@ final class PaymentFactory {
 
         return BraintreePaymentNonceProvider()
     }
+
+    func adyenEnvironment() -> Adyen.Environment {
```

Comment:
Is Production going to be handled in a future PR?

Reply:
exactly, I wasn't sure as there are multiple adyen production environments (US, Australia...) I'm guessing we will need to infer user locale to pick the right prod environment... might need to have a ticket for enabling prod... ?